### PR TITLE
Two small improvements to the updated SPV mining alert

### PR DESCRIPTION
**Pull request by saivann**

### CODE REVIEW

File: _alerts/2015-07-04-spv-mining.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
 update: 2015-07-06 02:00.  All times are UTC.</em></p>
 
 **Note: this alert is on-going: the situation has not yet been
-resolved. ([Update #1](#update-1))**
+resolved. ([Read more](#cause))**
```
Comment (harding) [inline _alerts/2015-07-04-spv-mining.html]: Here I kind of wanted to provide as easy way for people who had read previous versions of the alert to get straight to the new information.

Comment (saivann) [inline _alerts/2015-07-04-spv-mining.html]: Please feel free to ignore it, but FWIW, I didn't notice the paragraph due to (1) it being at the bottom of the page and (2) it not using any title, nothing very obvious.

Comment (harding) [inline _alerts/2015-07-04-spv-mining.html]: Maybe this is better: http://dg0.dtrt.org/en/alert/2015-07-04-spv-mining#update-1

018724d3a5d001ac8cb4c721e558366fb6be5a10

Comment (saivann) [inline _alerts/2015-07-04-spv-mining.html]: @harding Ah yes, probably!

Comment (harding) [inline _alerts/2015-07-04-spv-mining.html]: Cool.  I cherry picked your other commit, so closing this PR.  Thanks!

**Pull request closed without merging**